packages/pwa-kit-dev/src/utils/extensibility-utils.js

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (c) 2024, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
```

Comment:
Just curious: why new files in pwa-kit-react-sdk are written in typescript, while this one in pwa-kit-dev is not?

Reply:
No reason. Just happened like that because it was written first and this project is predominately js. When I wrote the TS interface in the SDK it was required to be in TS for obvious reasons.